WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -70,4 +72,23 @@ + (NSArray *)routes
   return FBResponseWithOK();
 }
 
++ (id<FBResponsePayload>)handleDismissKeyboardCommand:(FBRouteRequest *)request
+{
+  [request.session.application dismissKeyboard];
+  NSError *error;
+  BOOL isKbdNotPresent =
```

Comment:
Both work for me. Personally I find `isKeyboardPresent` easier to read (it is easy to skip not in the middle) , but if you prefer `isKeyboardNotPresent` I am fine with it.